server/controllers/recommendations/helpers/taskCompleteness.ts

```diff
@@ -107,6 +107,8 @@ export const taskCompleteness = (recommendation: RecommendationResponse, _featur
       recommendation.custodyStatus?.selected !== 'NO' || hasValue(recommendation.isMainAddressWherePersonCanBeFound),
   }
 
+  // const flagReviewAndSend = _featureFlags.flagReviewAndSend
```

Comment:
Should this be flagProbationAdmin?

Reply:
Good spot - it's commented out code anyway.  Can we relax this level of correctness please?  That would be fixed up pretty soon, when the real code goes in.  And re committing and running through CI will cost me about 40 minutes.

Reply:
Yes, CI takes too long and is quite painful at the moment. It ought to be 10 minutes at the most, but it's been allowed to extend way too long.  We really should pay some attention to it.
As regards the level of correctness, if we are going to leave in commented out code to ease introduction in the future, then it should be correct, or it doesn't make it easier in the future.  I would prefer no commented code at all, but have let that pass.